navit/xmlconfig.c

```diff
@@ -750,17 +762,31 @@ static void xinclude(xml_context *context, const gchar **attribute_names, const
     } else {
         dbg(lvl_debug,"expanding '%s'", href);
         we=file_wordexp_new(href);
-        we_files=file_wordexp_get_array(we);
+        we_files=file_wordexp_get_array(we);	/* Expand wildcards (if any) into a list of files */
         count=file_wordexp_get_count(we);
         dbg(lvl_debug,"%d results", count);
-        if (file_exists(we_files[0])) {
-            for (i = 0 ; i < count ; i++) {
-                dbg(lvl_debug,"result[%d]='%s'", i, we_files[i]);
-                doc_new.href=we_files[i];
-                parse_file(&doc_new, error);
+        for (i = 0 ; i < count ; i++) {
+            included_filename = g_strdup(we_files[i]);
+            if (*included_filename != '\0') { /* Non empty href */
+                if (*included_filename != '/') {	/* The filename's path is relative */
```

Comment:
On Windows, absolute paths may start with /, \ or drive letter name followed by a colon (like c: ).

Actually, It's a bit harder on Windows, as it keeps current working directory for each drive, but I doubt we should care about it here.

Reply:
@lains g_path_is_absolute should solve the problem.

Reply:
I am going to work on this. Thanks.

Reply:
This was submitted in:
https://github.com/navit-gps/navit/pull/559/commits/d09cb0cdb54f301afbfffa2dc054032867e8aab8
https://github.com/navit-gps/navit/pull/559/commits/093f5cbbf90e11eb753bc5beb7bb54253c132e85